.golangci.yml

```diff
@@ -0,0 +1,34 @@
+linters:
+  enable-all: true
+
+issues:
+  # Disable the default exclude list so that all excludes are explicitly
+  # defined in this file.
+  exclude-use-default: false
+
+  # Only report deltas off of main branch
+  new: true
+
+  exclude-rules:
+    # These pop up as a result of command tests and can be safely ignored. 
+    - linters: [errcheck]
+      text: 'Error return value of `(.*).Stop` is not checked'
+    - linters: [errcheck]
+      text: 'Error return value of `(.*).Write` is not checked'
+    - linters: [errcheck]
+      text: 'Error return value of `(.*).Remove` is not checked'
```

Comment:
I think these two could be useful for non-test code because, for example, there are a lot of things that implement  the [`Writer` interface](https://pkg.go.dev/io#Writer) and could be using that function. If this is something we really should ignore for tests, then maybe there's a way to exclude only test files? Btw, which test code is this fixing?